src/cargo/core/resolver/mod.rs

```diff
@@ -785,46 +785,51 @@ impl<'a> Context<'a> {
              dep: &Dependency) -> CargoResult<Vec<Candidate>> {
         let summaries = try!(registry.query(dep));
         summaries.into_iter().map(Rc::new).map(|summary| {
-            let mut replace = None;
-            let mut matched_spec = None;
-            for &(ref spec, ref dep) in self.replacements.iter() {
-                if !spec.matches(summary.package_id()) {
-                    continue
-                }
-
-                if replace.is_some() {
-                    bail!("overlapping replacement specifications found:\n\n  \
-                           * {}\n  * {}\n\nboth specifications match: {}",
-                          matched_spec.unwrap(), spec, summary.package_id());
-                }
+            // get around lack of non-lexical lifetimes
```

Comment:
Should I force--push? (I wrote that, as opposed to move it around, to be clear.)